src/wad/tex.rs

```diff
@@ -143,18 +154,18 @@ impl TextureDirectory {
     }
 
 
-    pub fn build_texture_atlas<'a, T: IntoIterator<Item = &'a WadName>>(
-            &'a self, names_iter: T) -> (TransparentImage, BoundsLookup) {
-        let entries = ordered_atlas_entries(&self.animated_walls,
-                                            |n| self.texture(&n),
-                                            names_iter);
+    pub fn build_texture_atlas<'a, T: IntoIterator<Item = &'a WadName>>
+                                                                        (&'a self,
+                                                                         names_iter: T)
+                                                                         -> (TransparentImage, BoundsLookup) {
+        let entries = ordered_atlas_entries(&self.animated_walls, |n| self.texture(&n), names_iter);
         let max_image_width = if let Some(width) = entries.iter().map(|e| e.image.width()).max() {
             width
-        } else  {
+        } else {
             return (TransparentImage {
-                        pixels: Vec::new(),
-                        size: Vec2::zero(),
-                    },
+                pixels: Vec::new(),
```

Comment:
This looks like the previous bug with the unaligned match branches in a tuple. Could you please revert this to the way it was indented before?